agent/xds/testdata/routes/api-gateway-with-http-route-and-inline-certificate.latest.golden

```diff
@@ -0,0 +1,31 @@
+{
+  "versionInfo":  "00000001",
+  "resources":  [
+    {
+      "@type":  "type.googleapis.com/envoy.config.route.v3.RouteConfiguration",
+      "name":  "8080",
+      "virtualHosts":  [
+        {
+          "name":  "api-gateway-listener-9b9265b",
+          "domains":  [
+            "*",
+            "*:8080"
+          ],
+          "routes":  [
+            {
+              "match":  {
+                "prefix":  "/"
```

Comment:
Had to double-check, but this is the expected behavior for defining a rule with only a Service backend (no matches or filters) 👍